index.html

```diff
@@ -9,11 +9,119 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<header id="header">
+    <img class ="logo"src="img/karma-logo.svg" alt="logo"  width="40px" height= "40px"/>
 
-</body>
-</html>
+<nav class ="navBar">
+    <a class="active" href="#Meet Karma">Meet Karma</a>
+    <a href="#How it Works">How it Works</a>
+    <a href="#Store">Store</a>
+    <a href="#Blog">Blog</a>
+    <a href="#Help">Help</a>
+    <a href="#Login">Login</a>
+      
+</nav>
+</header>
+<br><br><br><br><br>
```

Comment:
This is not how we create space between elements - apply a `margin-bottom`  to your header instead